src/cljc/ataru/schema/form_schema.cljc

```diff
@@ -455,6 +455,8 @@
    :notes                         s/Str
    :first-name                    (s/maybe s/Str)
    :last-name                     (s/maybe s/Str)
+   :hakukohde                     (s/maybe s/Str)
+   :state-name                    (s/maybe s/Str)
```

Comment:
Täs on nyt pakolliset avaimet ja vapaaehtoset arvot, kun ulos menevässä skeemassa on toisin päin. Ja kantsii varmaan tehä tosta `(apply s/enum review-states/hakukohde-review-type-names)` nimetty skeema kun sitä tarvitaan ainakin kolmessa paikassa.

Reply:
Niin. Laitoin vaan niin että sisään ei tuu dataa jos sitä ei oo, eli ei tuu nulleja, mutta kannasta palautuu kyllä aina noi arvot. Voishan se tietysti mennä toiseenkin suuntaan niin, turhaa koodia vaan parsia nil arvoja pois sieltä mäpistä.